images/operator-sdk/Dockerfile

```
@@ -19,6 +19,10 @@ RUN GOOS=linux GOARCH=$TARGETARCH make build/operator-sdk
 # Final image.
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
+ENV GO_VERSION 1.16.4
+
+RUN curl -O https://storage.googleapis.com/golang/go$GO_VERSION.linux-386.tar.gz
+
 RUN microdnf install -y golang make which
```

Gotta unpack the thing, also I don't think that's the recommended DL link on the go website.
```suggestion
ARG TARGETARCH

RUN microdnf install -y make which tar gzip

ENV GO_VERSION 1.16.4
RUN curl -sSLo /tmp/go.tar.gz https://golang.org/dl/go${GO_VERSION}.linux-${TARGETARCH}.tar.gz \
	&& rm -rf /usr/local/go \
	&& tar -C /usr/local -xzf /tmp/go.tar.gz \
	&& ln -sf /usr/local/go/bin/* /usr/local/bin/ \
	&& rm -f /tmp/go.tar.gz \
	&& go version
```

---

Done. Thank you.